src/services/sendEmail.js

```diff
@@ -15,8 +13,9 @@ const createVerificationEmail = hash => {
   const inviteUrl = `${CLIENT_URL}/verify/${hash}`;
 
   return {
-    from: 'hongyichen@uwblueprint.org',
+    from: 'info@raisingtheroof.org',
     subject: `Verify your email for Raising the Roof's Toque Campaign`,
+    replyTo: 'skovacev@raisingtheroof.org',
     html: `Thank you for signing up! Please verify your email with this link: <a href="${inviteUrl}">${inviteUrl}</a>`,
```

Comment:
Not sure if we want to hyperlink "here" like we do with resetpassword, just to keep user experience the same :)

i.e. Thank you for signing up! Please verify your email <a href="${inviteUrl}"> here </a>

Reply:
fixed in b3bfe6c